packages/api/amplify_api/test/graphql_error_test.dart

```diff
@@ -68,6 +68,6 @@ void main() {
         .response;
```

Comment:
request: now that we are changing the expected logic for errors, could you please test that a happy case response has null errors? You could just add an assertion on one of the query tests or add another test.